src/client/components/informationPanel/InformationSocials.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Text, Card, theme, Button, IconClose } from '@aragon/ui'
+import {
+  Social,
+  Link,
+  IconGitHub,
+  IconTwitter,
+  IconVerified,
+} from '../styled-components'
+
+const PopoverCard = ({ social, setPopover }) => {
+  return (
+    <VerifyCard>
+      <Text.Block size="xlarge">Verify my {social}</Text.Block>
+      <CardCloseButton
+        type="button"
+        onClick={() => {
+          setPopover('')
+        }}
+      >
+        <IconClose />￼
+      </CardCloseButton>
+
+      <Text.Block size="small">
+        Aragon manages profiles using 3box.io. To verify your {social}, you must
+        visit your 3box profile.
+      </Text.Block>
+      <Link size="small" href="https://3box.io">
+        Take me to my 3box
+      </Link>
+    </VerifyCard>
+  )
+}
+
+PopoverCard.propTypes = {
+  social: PropTypes.string,
+  setPopover: PropTypes.func.isRequired,
```

Comment:
I suggest we rename this component and its props:

```js
VerifySocial.propTypes = {
  account: PropTypes.string.isRequired,
  onClose: PropTypes.func.isRequired,
```

Reply:
Good idea. As said before, naming variables is a fine art ;) However, I left:
  const [activePopover, setPopover] = useState('')
in InformationCard, in case there is need for popovers of different kind than "verify social" in the future.

Reply:
> in case there is need for popovers of different kind than "verify social" in the future.

Sounds like premature optimization to me! 😉